src/_pytest/capture.py

```diff
@@ -413,30 +415,27 @@ def safe_text_dupfile(f, mode, default_encoding="UTF8"):
 class EncodedFile:
     errors = "strict"  # possibly needed by py3 code (issue555)
```

Comment:
A bit strange that it says `strict` here but actually goes to use `replace`.

Reply:
Yeah, likely an oversight.  Ref to issue: https://bitbucket.org/pytest-dev/pytest/issues/555
Will create a followup issue.